libraries/adsk/shaders/wood/parquet_3/ND_parquet_03_shader_impl.mtlx

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <materialx version="1.38">
    <!-- Sample parquet shader implementation template -->
-   <nodegraph name="NG_parquet_03_shader" namespace="adsk">
+   <nodegraph name="NG_parquet_03_shader_template" namespace="adsk">
```

Comment:
The nodegraph for the nodedef has the same name as this nodegraph. I did a copy and paste
to extract out the nodegraph but forgot to rename it.